test/DynamoCoreWpfTests/GraphLayoutTests.cs

```diff
@@ -32,13 +32,7 @@ public void GraphLayoutEmpty()
 
             ViewModel.DoGraphAutoLayout(null);
 
-            Assert.IsNull(ViewModel.CurrentSpace.LayoutGraph);
```

Comment:
The unit tests were checking for the layering information from the combined graph object `Model.LayoutGraph`. This now has been changed to `Model.LayoutSubgraphs` where each subgraph has its own layering information.